### ISSUE TITLE
Android 64-bit ARMv8 build

### ISSUE THREAD

**Issue by atouchet** [servo/servo#23691]
[Google is requiring that new Android apps on the Play store must have a 64-bit version starting August 1, 2019.](https://android-developers.googleblog.com/2019/01/get-your-apps-ready-for-64-bit.html) Servo for Android is still built as a 32-bit ARMv7 app. Are there plans to build Servo for Android targeting 64-bit ARMv8 and what work would that take to do so? I think it would be a good idea to have a plan to move to 64-bit as it's clear that the Android ecosystem is moving away from ARMv7.

**Comment by jdm** [comment]
I think this should be straightforward. It definitely requires changing the [default android target](https://github.com/servo/servo/blob/master/python/servo/command_base.py#L313) to `aarch64-linux-android`, and adding to the list in https://github.com/servo/servo/blob/master/support/android/apk/jni/Application.mk#L5. We may need to make an android gstreamer bundle for the new target as well based on https://github.com/servo/libgstreamer_android_gen/, but that may already exist. It may just be a matter of seeing if anything breaks; all of the build files I've looked at already have arm64 configurations.

**Comment by zhuowei** [comment]
I tried this: I only had to make one change ([fix the gcc-toolchain flag passed to clang](https://github.com/zhuowei/servo/commit/4bd25280e628a5c7c8e9084c613f7bdf0a8bf91d)), then running `./mach build --release --target aarch64-linux-android` just worked and I was able to get [an APK](https://drive.google.com/file/d/1hxjkyP9XR-_Ur4nuXrP-S9xAcV1ngPAf/view?usp=sharing).

Unfortunately, the APK doesn't seem to do anything. It opens but I can't seem to get it to load a url. I'll have to run `adb logcat` to see what's going on.
